components/engine_traits/src/engine_util/metrics_flusher.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Seems the submodule  `engine_util` is also unnecessary.

Reply:
removed.